.github/workflows/run-linting.yml

```diff
@@ -0,0 +1,17 @@
+name: Run Linter
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+      - run: npm ci
+      - run: npm run lint
```

Comment:
Do we want to make the workflow fail with --max-warnings? It's super sweet we get "check annotations" even without the workflow failing

Reply:
Sure, I can do that!